initramfs/init.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/busybox sh
+#
+# SPDX-License-Identifier: AGPL-3.0+
+
+set -eu
+
+ROOT_MOUNT="/mnt/root"
+BB_BIN="/bin/busybox"
+
+init="/sbin/init"
+ro=""
+root=""
+rootflags=""
+rootfstype="auto"
+
+shutdown()
```

Comment:
why have a seperate shutdown, don't we always want to reboot or end up in the rescue shell?
then just have a reboot that if unsucessful does this?

Reply:
answer is to avoid reboot loops